app/components/ui/toast.tsx

```diff
@@ -75,7 +75,7 @@ const ToastClose = React.forwardRef<
   <ToastPrimitives.Close
     ref={ref}
     className={cn(
-      'absolute top-2 right-2 rounded-md p-1 text-foreground/50 opacity-0 transition-opacity hover:text-foreground focus:opacity-100 focus:outline-none focus:ring-2 group-hover:opacity-100 group-[.destructive]:text-red-300 group-[.destructive]:focus:ring-red-400 group-[.destructive]:focus:ring-offset-red-600 group-[.destructive]:hover:text-red-50',
```

Comment:
what happened here?

Reply:
ah yes this is their design choice. i see it on their demos as well

Reply:
btw should we maybe completely hide that x on the mobile devices?

that might be the reason they show it on hover only becuase hover is not possible on mobile and x is not needed there